MoviesMVVM/MoviesMVVM/Proxy/Proxable .swift

```diff
@@ -0,0 +1,9 @@
+// Proxable .swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Протокол для скачивания фото из интернета
+protocol Proxable {
```

Comment:
```suggestion
protocol Proxable {
```

странное название протокола для скачивания фото из интернета. по нему вообще не понятно че он делает

Reply:
по названию понятно понятно что это протокол Прокси